net-mgmt/pfSense-pkg-snmptt/files/usr/local/pkg/snmptt.xml

```diff
@@ -0,0 +1,730 @@
+			<fieldname>multiple_event</fieldname>
+			<description>
+				<![CDATA[
+					Set to 1 to allow multiple trap definitions to be executed for the same trap.<br/>
+					Set to 0 to have it stop after the first match.<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>1</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>DNS Enable</fielddescr>
+			<fieldname>dns_enable</fieldname>
+			<description>
+				<![CDATA[
+					Set to 0 to disable DNS resolution.<br/>
+					Set to 1 to enable DNS resolution.<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>0</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Strip Domain</fielddescr>
+			<fieldname>strip_domain</fieldname>
+			<description>
+				<![CDATA[
+					Set to 0 to enable the use of FQDN (Fully Qualified Domain Names). If a host name is<br/>
+					passed to SNMPTT that contains a domain name, it will not be altered in any way by<br/>
+					SNMPTT. This also affects resolve_value_ip_addresses.<br/>
+					Set to 1 to have SNMPTT strip the domain name from the host name passed to it. For<br/>
+					example, server01.domain.com would be changed to server01<br/>
+					Set to 2 to have SNMPTT strip the domain name from the host name passed to it<br/>
+					based on the list of domains in strip_domain_list.<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>0</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+				<option><name>2</name><value>2</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Strip Domain List</fielddescr>
+			<fieldname>strip_domain_list</fieldname>
+			<encoding>base64</encoding>
+			<type>textarea</type>
+			<rows>5</rows>
+			<cols>50</cols>
+			<description>List of domain names that should be stripped when strip_domain is set to 2.</description>
+		</field>
+		<field>
+			<fielddescr>Resolve Value IP Addresses</fielddescr>
+			<fieldname>resolve_value_ip_addresses</fieldname>
+			<description>
+				<![CDATA[
+					Set to 0 to disable resolving ip address to host names.<br/>
+					Set to 1 to enable resolving ip address to host names.<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>0</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Net SNMP Perl Enable</fielddescr>
+			<fieldname>net_snmp_perl_enable</fieldname>
+			<description>
+				<![CDATA[
+					Set to 1 to enable the use of the Perl module from the UCD-SNMP / NET-SNMP package.<br/>
```

Comment:
Put short descriptions of these behaviors in the "name" part of the options list.